src/detect-bsize.c

```diff
@@ -65,6 +65,8 @@ void DetectBsizeRegister(void)
 #define DETECT_BSIZE_RA 2
 #define DETECT_BSIZE_EQ 3
 
+const char *bsize_mode_strings[] = { "<", ">", "<>", "=" };
```

Comment:
Is this variable used ?

PS : could we reuse stuff from detect-engine-uint.h here ?

Reply:
It's not used -- will remove.

Reply:
Ok.
So, could we reuse stuff from detect-engine-uint.h here ?